keps/sig-instrumentation/4785-resource-state-metrics/README.md

```diff
@@ -0,0 +1,1216 @@
+<!--
+**Note:** When your KEP is complete, all of these comment blocks should be removed.
```

Comment:
^ are we good to remove all comment blocks? That will make it easier to review I guess? @rexagod

Reply:
I believe "complete" here means stable graduation, but I could be wrong.